### ISSUE TITLE
Error does not show stderr when using ansible-galaxy install even with verbose

### ISSUE THREAD

**Issue by waqark3389** [ansible/galaxy#2212]
## Feature Request
Background:

When running:

`ansible-galaxy install -vvvv -r requirements.yml -p roles`

requirements.yml file:

```
- src: git+https://<redacted>.<redacted>co.uk/scm/anr/ansible-os-hardening
  version: master
- src: git+https://<redacted>.<redacted>co.uk/scm/waq/amazon2-hardening
  version: master
- src: geerlingguy.ruby

```

This is the error presented:

```
Processing role ansible-os-hardening 
archiving ['/usr/bin/git', 'archive', '--prefix=ansible-os-hardening/', '--output=/home/jenkins/.ansible/tmp/ansible-local-15v_mf6qp9/tmpuemih8ng.tar', 'master']
- extracting ansible-os-hardening to /home/centos/slave/workspace/-Images_amazonlinux2-gold_master/roles/ansible-os-hardening
- ansible-os-hardening (master) was installed successfully
Processing role amazon2-hardening 
[WARNING]: - amazon2-hardening was NOT installed successfully: - command
/usr/bin/git clone
https://<redacted>.<redacted>co.uk/scm/waq/amazon2-hardening
amazon2-hardening failed in directory /home/jenkins/.ansible/tmp/ansible-
local-15v_mf6qp9/tmp8jpsgl_v (rc=128)
ERROR! - you can use --ignore-errors to skip failed roles and finish processing the list.
```

### Use Case
Get to see error
### Proposed Solution
Add a better error message
### Alternatives
Ran manually and it works from same machine etc. Just not within ansible-galaxy
### Implementation
Show error message including stderr of git command

**Comment by chouseknecht** [comment]
This needs to be filed in [https://github.com/ansible/ansible](https://github.com/ansible/ansible).

**Comment by s-hertel** [comment]
@waqark3389 FYI, this should be fixed for 2.10 by https://github.com/ansible/ansible/pull/66006.

**Comment by waqark3389** [comment]
Sweet - thanks :)
